script/phase-1b/03-UsdsSkyFarmingCheck.s.sol

```diff
@@ -54,6 +58,9 @@ contract Phase1b_UsdsSkyFarmingCheckScript is Script {
         require(DssVestWithGemLike(vest).res(vestId) == 1, "DssVest/invalid-vest-res");
         require(DssVestWithGemLike(vest).usr(vestId) == dist, "DssVest/wrong-dist");
         require(DssVestWithGemLike(vest).mgr(vestId) == address(0), "DssVest/mgr-should-not-be-set");
+        require(DssVestWithGemLike(vest).bgn(vestId) == vestBgn, "DssVest/invalid-bgn");
```

Comment:
You are missing chek for ` Vest.cap`

Reply:
Saw you add `file(CAP)` to the `UsdsSkyFarmingInitSpell `

Reply:
This is out of the scope of this repo.
Cap is a system parameter.

Reply:
Nvm, saw your comments on the issue